tools/pipeline_perf_test/test_suites/integration/templates/configs/otelcol/otap-filter-otap.yaml

```diff
@@ -0,0 +1,60 @@
+receivers:
+  otelarrow:
+    protocols:
+      grpc:
+        endpoint: 0.0.0.0:4317
+
+
+exporters:
+  debug:
```

Comment:
we don't need debug exporter here. (its not used in pipeline, so can be removed.)